nbval/plugin.py

```diff
@@ -97,6 +97,12 @@ def pytest_addoption(parser):
                     type=float,
                     help='Timeout for kernel startup, in seconds.')
 
+    group.addoption('--sanitize-with',
+                    help='(deprecated) Alias of --nbval-sanitize-with')
+
+    group.addoption('--current-env', action='store_true',
+                    help='(deprecated) Alias of --nbval-current-env')    
+
```

Comment:
Could we not add a `dest` to these, so that the later code only needs to consider one value?

Reply:
It is certainly a shame to have so much code dedicated to option handling, and for the history to be visible in more than one place (at least for a while), but I don't know what else to do...